### ISSUE TITLE
Error and questions regarding command line usage

### ISSUE THREAD

**Issue by enarkay** [amrshaheen61/UE4LocalizationsTool#35]
I wish to batch export and import from the unpacked assets.
I tried what I though was the correct use of the command line, but am running into an error.

Could you take a look at what I maybe doing wrong, and how to fix it?
Below is the command I ran and a screenshot of the cmd window with the error.

UE4localizationsTool_v2.2.exe exportall "E:\Localization\Unreal Engine Tools Collection\Fmodel\Output\Exports\Swallow\Content" text.txt

![image](https://github.com/amrshaheen61/UE4LocalizationsTool/assets/7910166/8e17a044-232d-4939-a1dd-ba58bc5421d6)

I've done some googling to find the English version of the error message I received, and apparently it should translate to the below:

The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter variable name: left
Actual value is 161.

**Comment by amrshaheen61** [comment]
@enarkay 
what about this?
[UE4localizationsTool.zip](https://github.com/amrshaheen61/UE4LocalizationsTool/files/11774791/UE4localizationsTool.zip)

**Comment by enarkay** [comment]
Thank you very much I think that solved the problem perfectly!

While awaiting for your reply, I did some trials and errors to find out that it was the sound files that were throwing off the errors. So while I did manage to kind of manually solve the problem by removing the sound files from target folder, I was wondering if there was a better solution since having to manually locate and remove sound files or other exceptions could turn out to be quite the labor.

I see that you made exceptions to be just ignored and the program to pass on to next files - it works like a charm, thank you very much :)
